public/js/edudig.js

```diff
@@ -0,0 +1,8 @@
+function flagQR(e) {document.getElementsByClassName('grid-item-info')[0].innerHTML='QR flag'}
```

Comment:
Here I would probably do _one_ function `function setFlag(flagName) { document.getElementsByClassName('grid-item-info')[0].innerHTML=flagName}` which can then in the HTML be used like `setFlag("AA flag")`